locations/ska95.yml

```diff
@@ -48,6 +53,11 @@ networks:
     assignments:
       ska95-core: 1
 
+  - vid: 41 # Private LAN where private Internet is connected
+    role: ext
```

Comment:
This is called `uplink` now

Reply:
But then i'd question the name. why should it be uplink when there is no uplink?

Reply:
I always thought the purpose of the `ext` role was to attach an uplink network, usually DSL or mobile. I've never seen it used without a tunnel

Reply:
i was probably the only one who ever has used it. :)

Reply:
It's a legit use case, it's just news to me :-)

Reply:
ok, so the ext changes / fixes are fine? can we merge this?